### PR TITLE
Add user_op interpretation

### LINKED ISSUE #9310: Transaction interpretation/summary: support User Operations
similar to .../summary method for transactions we want to create summary method for user operations

### PR DESCRIPTION
**Pull request by nikitosing**
Closes #9310 

## Changelog
- Add `/api/v2/proxy/account-abstraction/operations/{operation_hash_param}/summary` endpoint
- Add `decoded_call_data` field to `/api/v2/proxy/account-abstraction/operations/{operation_hash_param}` response
- Set default value of `query_decompiled_code_flag` to false in `hash_to_address/2` in Explorer.Chain

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [ ] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/controllers/api/v2/fallback_controller.ex [hunk 1]
```diff
@@ -32,6 +32,7 @@ defmodule BlockScoutWeb.API.V2.FallbackController do
   @empty_response "Empty response"
   @tx_interpreter_service_disabled "Transaction Interpretation Service is not enabled"
```
Comment (vbaranov) [inline apps/block_scout_web/lib/block_scout_web/controllers/api/v2/fallback_controller.ex:33]:
```suggestion
  @tx_interpreter_service_disabled "Transaction Interpretation Service is disabled"
```

**Review by vbaranov — COMMENTED** [review]
This is not part of this concrete PR, but let's process the case when `MICROSERVICE_ACCOUNT_ABSTRACTION_ENABLED=true` and no `MICROSERVICE_ACCOUNT_ABSTRACTION_URL` is set, anyway. Currently, it throws an error:
```
2024-03-04T10:38:13.227 [error] #PID<0.2814.0> running BlockScoutWeb.Endpoint (connection #PID<0.2813.0>, stream id 1) terminated
Server: localhost:4000 (http)
Request: GET /api/v2/proxy/account-abstraction/operations/0xcb560d77b0f3af074fa05c1e5c691bcdfe457e630062b5907e9e71fc74b2ec61
** (exit) an exception was raised:
    ** (FunctionClauseError) no function clause matching in String.ends_with?/2
        (elixir 1.14.5) lib/string.ex:2395: String.ends_with?(nil, "/")
        (explorer 6.2.0) lib/explorer/utility/microservice.ex:8: Explorer.Utility.Microservice.base_url/2
        (explorer 6.2.0) lib/explorer/microservice_interfaces/account_abstraction.ex:201: Explorer.MicroserviceInterfaces.AccountAbstraction.base_url/0
        (explorer 6.2.0) lib/explorer/microservice_interfaces/account_abstraction.ex:157: Explorer.MicroserviceInterfaces.AccountAbstraction.operation_by_hash_url/1
        (explorer 6.2.0) lib/explorer/microservice_interfaces/account_abstraction.ex:18: Explorer.MicroserviceInterfaces.AccountAbstraction.get_user_ops_by_hash/1
        (block_scout_web 6.2.0) lib/block_scout_web/controllers/api/v2/proxy/account_abstraction_controller.ex:19: BlockScoutWeb.API.V2.Proxy.AccountAbstractionController.operation/2
        (block_scout_web 6.2.0) lib/block_scout_web/controllers/api/v2/proxy/account_abstraction_controller.ex:1: BlockScoutWeb.API.V2.Proxy.AccountAbstractionController.action/2
        (block_scout_web 6.2.0) lib/block_scout_web/controllers/api/v2/proxy/account_abstraction_controller.ex:1: BlockScoutWeb.API.V2.Proxy.AccountAbstractionController.phoenix_controller_pipeline/2
        (phoenix 1.5.14) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
        (phoenix 1.5.14) lib/phoenix/router/route.ex:41: Phoenix.Router.Route.call/2
        (phoenix 1.5.14) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
        (block_scout_web 6.2.0) lib/block_scout_web/endpoint.ex:1: BlockScoutWeb.Endpoint.plug_builder_call/2
```

The same also applied to missing `MICROSERVICE_TRANSACTION_INTERPRETATION_URL` with enabled AA service. Let's fix this as well.

File: apps/block_scout_web/lib/block_scout_web/controllers/api/v2/proxy/account_abstraction_controller.ex [hunk 2]
```diff
@@ -20,6 +20,31 @@ defmodule BlockScoutWeb.API.V2.Proxy.AccountAbstractionController do
     |> process_response(conn)
   end
 
+  def summary(conn, %{"operation_hash_param" => operation_hash_string, "just_request_body" => "true"}) do
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/controllers/api/v2/proxy/account_abstraction_controller.ex:23]: Please, add doc and spec

File: apps/block_scout_web/lib/block_scout_web/microservice_interfaces/transaction_interpretation.ex [hunk 3]
```diff
@@ -17,26 +18,52 @@ defmodule BlockScoutWeb.MicroserviceInterfaces.TransactionInterpretation do
   @api_true api?: true
   @items_limit 50
 
-  @spec interpret(Transaction.t()) ::
+  @doc """
+  Interpret transaction or user operation
+  """
+  @spec interpret(Transaction.t() | map()) ::
           {{:error, :disabled | binary()}, integer()}
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/microservice_interfaces/transaction_interpretation.ex:25]:
```suggestion
  """
  @spec interpret(Transaction.t() | map(), (Transaction.t() -> any()) | (map() -> any())) ::
          {{:error, :disabled | binary()}, integer()}
```

**Review by sl1depengwyn — APPROVED** [review]

**Review by vbaranov — APPROVED** [review]

**Pull request merged**